js/tsconfig.base.esm.json

```diff
@@ -1,7 +1,9 @@
 {
-  "extends": "./tsconfig.base.es5.json",
+  "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "module": "ES6",
-    "moduleResolution": "node"
+    "target": "ES2020",
+    "moduleResolution": "node10",
```

Comment:
I think I'm okay with using the last supported version for node since this is greenfield.